src/main/c/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-file(GLOB SOURCES Vips.c VipsContext.c VipsException.c VipsImage.c)
+file(GLOB SOURCES Vips.c VipsContext.c VipsException.c VipsImage.c Max1Result.c)
 
 add_library(JVips SHARED ${SOURCES})
 add_dependencies(JVips Vips_header VipsContext_header VipsImage_header)
```

Comment:
You have to add a dependency on Max1Result_header too

Reply:
done